src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -0,0 +1,166 @@
+    color: var(--base-6);
+    margin: 1em;
+    max-width: 94%;
+    width: 600px;
+}
+
+.modal-wrap .modal-header {
+    background: var(--table-header);
+    padding: 15px 30px;
+}
+.modal-wrap .modal-header h3 {
+    color: var(--base-6);
+    margin: 0;
+}
+
+.modal-wrap .modal-header .close-modal {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+}
+
+.modal-wrap .modal-header .close-modal:hover { opacity: 1; }
+
+.modal-wrap .modal-content {
+  background: var(--base-1);
+  margin: 0;
+  padding: 15px 30px;
+  width: 100%;
+  z-index: 100000;
+}
+.modal-content h4 {
+    border-bottom: var(--border);
+    margin: 0 0 15px;
+    padding: 0 0 5px;
+}
+.modal-content input, .modal-content .form-help {
+    margin-left: 15px;
+}
+.modal-content label {
+    cursor: pointer;
+    font-size: 16px;
+    margin-left: 3px;
+}
+.modal-content .form-help {
+    color: var(--color-muted);
+    font-size: 14px;
+    margin: 5px 0 15px 33px;
+}
+.modal-content .form-help + h4 {
+    margin-top: 45px;
+}
+
+.modal-footer {
+    background: var(--base-0);
+    padding: 15px 30px;
+    text-align: right;
+    z-index: 100000;
+}
+</style>
+
+<a href="#" id="open-settings" data-triger="profiler-settings">Settings</a>
+
+<div class="modal-wrap" id="profiler-settings">
+    <div class="modal-container">
+        <div class="modal-header">
+            <h3>Configuration Settings</h3>
+        </div>
+
+        <div class="modal-content">
+            <h4>Theme</h4>
+
+            <input class="config-option" type="radio" name="theme" value="light" id="settings-theme-light">
+            <label for="settings-theme-light">Light</label>
+            <p class="form-help">Default theme. Improves readability.</p>
+
+            <input class="config-option" type="radio" name="theme" value="dark" id="settings-theme-dark">
+            <label for="settings-theme-dark">Dark</label>
+            <p class="form-help">Reduces eye fatigue. Ideal for low light conditions.</p>
+
+            <h4>Page Width</h4>
+
+            <input class="config-option" type="radio" name="width" value="light" id="settings-width-normal">
+            <label for="settings-width-normal">Normal</label>
+            <p class="form-help">Fixed page width. Improves readability.</p>
+
+            <input class="config-option" type="radio" name="width" value="dark" id="settings-width-wide">
+            <label for="settings-width-wide">Ultra-wide</label>
+            <p class="form-help">Dynamic page width. Same width as the browser window.</p>
+        </div>
+
+        <div class="modal-footer">
+            <button class="btn close-modal">Save changes</button>
+        </div>
+    </div>
+</div>
+
+<script>
+    let configOptions = document.querySelectorAll('.config-option');
+    let oppositeValues = { 'light': 'dark', 'dark': 'light', 'normal': 'wide', 'wide': 'normal' };
+    [...configOptions].forEach(option => {
+        option.addEventListener('click', function (event) {
```

Comment:
you should rely on the `change` event. I'm not sure changes performed using the keyboard are triggering `click` events.